modules/apps/frontend-theme/frontend-theme-classic/src/css/_custom.scss

```diff
@@ -1,4 +1,3 @@
-// @import 'components/info-panel'; // To Verify
```

Comment:
Not relevant change

![info-panel-2](https://user-images.githubusercontent.com/46778114/107658327-4ee54d80-6c86-11eb-99a3-eb106c9b63f5.jpg)


![info-panel-1](https://user-images.githubusercontent.com/46778114/107658308-4ab93000-6c86-11eb-9ced-08d1d63893bf.jpg)